lib/grpc.dart

```diff
@@ -36,6 +36,7 @@ export 'src/client/options.dart'
         BackoffStrategy,
         defaultBackoffStrategy,
         ChannelOptions;
+export 'src/client/proxy.dart' show Proxy;
```

Comment:
nit: If `Proxy` is a public class then it should probably have some doc comments. 

Also I just noticed that the file is missing copyright header. (Maybe double check that all newly created files have copyright headers).

Reply:
Time to enable https://github.com/dart-lang/ecosystem/blob/main/.github/workflows/health.yaml in this repo...